projects/app/workbench-application-platform/testing-app/src/app/testing/ping-intent/ping-intent-panel.component.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2018 Swiss Federal Railways
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ *  SPDX-License-Identifier: EPL-2.0
+ */
+
+import {Component, OnDestroy} from '@angular/core';
+import {NilQualifier} from '@scion/workbench-application-platform.api';
+import {first, takeUntil} from 'rxjs/operators';
+import {Observable, Subject} from 'rxjs';
+import {IntentService} from '@scion/workbench-application.core';
+
+@Component({
+  selector: 'app-ping-intent-panel',
+  templateUrl: './ping-intent-panel.component.html',
+  styleUrls: ['./ping-intent-panel.component.scss'],
+})
+export class PingIntentPanelComponent implements OnDestroy {
+
+  public result$: Observable<string>;
+  private _destroy$ = new Subject<void>();
+
+  constructor(private _intentService: IntentService) {
+  }
+
+  public onPing(message: string): void {
+    this.result$ = this._intentService.issueIntent$<string>('ping', NilQualifier, message)
+      .pipe(
+        first(),
+        takeUntil(this._destroy$)
```

Comment:
The `takeUntil` operator is not necessary if you let Angular subscribe to the `Observable` via `async` pipe (because it unsubscribes itself when the template gets destroyed). Also, please remove the `_destroy$` Subject and `OnDestroy` lifecycle hook. 
According to our project code style, if only having a single RxJS operator, we write it on the same line as the pipe method itself.
`this.result$ = this._intentService.issueIntent$<string>('ping', NilQualifier, message)\n.pipe(first())`